.travis.yml

```diff
@@ -1,15 +1,12 @@
 language: node_js
 node_js:
-  - "4.2.2"
+  - "4.4.2"
 
 services:
   - redis-server
 
-cache:
-  directories:
-    - node_modules
-
```

Comment:
Have you seen what we are doing in lev-web? - https://github.com/UKHomeOffice/lev-web/blob/master/.travis.yml

Basically, we are caching but blowing away just node-sass as this is the thing that is locked to the node version.

Why did you make the change to stop caching anyway? (You're only testing against one node version anyway.)